File: front/interfaces/stashCampaign.ts
interface StashCampaign {
  campaign_id: string;
  description: string;
  campaign_creator: string;
  reward: number;
  reward_token: string;
  campaign_type: number;
  max_submissions: number;
  remained_submissions: number;
  token_symbol: string;
  blockchain: string;
  top_left: {
    lat: number;
    long: number;
  };
  bottom_right: {
    lat: number;
    long: number;
  };
}

File: backend/app/schemas/stash_campaign.py
from pydantic import BaseModel

class Point(BaseModel):
    lat: float
    long: float

class StashCampaign(BaseModel):
    campaign_id: str
    description: str
    campaign_type: int
    campaign_creator: str
    reward: str
    reward_token: str
    token_symbol: str
    blockchain: str
    max_submissions: int
    remained_submissions: int
    top_left: Point
    bottom_right: Point
